src/components/components/ebay-notice-base/index.marko

```diff
@@ -53,16 +54,35 @@ $ const {
                     class=iconClass
                 />
             </else-if>
-            <else-if(status === 'education' && type === 'section')>
+            // deprecated status === education. We should probably remove this whole icon on the next major version
```

Comment:
Not sure what you are trying to say here, users can add whatever they want here.

Reply:
The footer should only have a CTA; users should not be able to do whatever they want. I simply included a reference to the markup to demonstrate how it should be structured. `education-notice` is a lot more explicit how the sections should work and what they should have.

Reply:
For now, will leave it as is. 
There could be multiple ways a person would add a CTA, like they might want to add their own custom component for it, or use fake-link etc. This also requires us to handle all those events on the CTA like focus, click, link etc. 
We have some places where we have it more explicit if its simple events, but for more complex interactions we allow users to pass in their component.

Reply:
If we do that, how can we ensure the styling from DS remains intact? What if they use a custom component that looks completely different and unsupported by DS and Skin? We're ok with the way things would look? Can't we just have a link with an href or a button that emits the click and the dev does whatever they want with the click?

Reply:
We can, but we also need to allow them to change it to a link or button. If they want it to be a link, it should have href, otherwise we need to emit a click event. 
This is what the code looks like in both cases:
Current
```
<ebay-education-notice>
   <@footer><ebay-fake-link on-click("trigger")>Link</></@footer>
</>
``` 

Proposed
```
<ebay-education-notice on-cta-button-click('trigger')>
   <@cta as="button" class="fake-link">Link</@cta>
</> 
```

I don't like how they have to add fake-link class if they set it to button personally. I think its better if they pass in a component.

Reply:
I see. In that case, can we add the fact that the component requires either an `<a>` or `<ebay-fake-link>` to look as it was intended. We're currently not enforcing those elements and nowhere that I can are we proving guidance for what it needs.